src/components/Gradebook/gradebook.scss

```diff
@@ -42,6 +42,25 @@
         margin-left: 10px;
     }
 }
+.grade-history-header{
+    float: left;
+}
+
+.grade-history-assignment{
+    padding-right: 49px;
```

Comment:
NIT: These class names are super misleading and we would be better suited with more utility classes such as a pr-49 or something like that. That said, this is fine for now and can be refactored later.